src/ImageSharp/Formats/Tiff/TiffBitsPerSample.cs

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Six Labors.
+// Licensed under the Apache License, Version 2.0.
+
+namespace SixLabors.ImageSharp.Formats.Tiff
+{
+    /// <summary>
+    /// The number of bits per component.
+    /// </summary>
+    public enum TiffBitsPerSample
```

Comment:
I think we should change the naming here. Look how we do it with Png.

https://github.com/SixLabors/ImageSharp/blob/9959d51795bab005aa3c793126ab35831fc46923/src/ImageSharp/Formats/Png/PngBitDepth.cs#L10-L36

Reply:
Renamed with 966d743d

Reply:
That's `TiffBitsPerPixel` (_which I actually hadn't spotted!_ 👍 ), we need `TiffBitsPerSample` also..

Reply:
Dont know how i managed to overlook that, its changed now with 22e92ade

Reply:
can it make like that?

```
public enum TiffBitsPerSample
{
	Bit1 = 1,
	Bit4 = 4,
	Bit8 = 8,
	Bit24 = 24
}
```

Reply:
yes makes sense, changed with: 6bb1c809